### PR TITLE
Modify perps market configurations for Stagnet

### PR DESCRIPTION
**Pull request by witgaw**

### CODE REVIEW

File: proposals/market_perp_btcusd.go [hunk 1]
```diff
@@ -41,12 +41,12 @@ func NewBTCUSDPerpetualMarketProposal(
...
-						PositionDecimalPlaces:   3,
-						LinearSlippageFactor:    "0.1",
-						QuadraticSlippageFactor: "0.1",
+						PositionDecimalPlaces:   4,
+						LinearSlippageFactor:    "0.001",
+						QuadraticSlippageFactor: "0.0",
 						Instrument: &vega.InstrumentConfiguration{
 							Name: name,
-							Code: "BTCUSD.MF21",
+							Code: "BTCUSD.MF21", //TODO (WG): What's the meaning of MF21??
```
Comment (jeremyletang) [inline proposals/market_perp_btcusd.go:49]: I guess we could change this to BTCUSD.PERP? Or whatever

Comment (davidsiska-vega) [reply proposals/market_perp_btcusd.go:49]: Yeah, MF21 is some leftover from 2021. Best forget about it.

File: proposals/market_perp_btcusd.go [hunk 2]
```diff
@@ -131,40 +131,45 @@ func NewBTCUSDPerpetualMarketProposal(
...
-							TriggeringRatio:  "0.0",
+							TriggeringRatio:  "0.7",
 							AuctionExtension: 1,
 						},
 						RiskParameters: &vega.NewMarketConfiguration_LogNormal{
 							LogNormal: &vega.LogNormalRiskModel{
-								RiskAversionParameter: 0.0001,
-								Tau:                   0.0000190129,
+								RiskAversionParameter: 0.000001,
+								Tau:                   0.0001140771161,
```
Comment (davidsiska-vega) [inline proposals/market_perp_btcusd.go:168]: I'd be up for a more funky `tau`; let's aim for like a 2 minute tau so we get proper degen level leverage.

File: proposals/market_perp_btcusd.go [hunk 3]
```diff
@@ -131,40 +131,45 @@ func NewBTCUSDPerpetualMarketProposal(
...
-									Horizon:          300,
+									Horizon:          3600,
 									Probability:      "0.9999",
-									AuctionExtension: 60,
+									AuctionExtension: 120,
 								},
 							},
 						},
-						LpPriceRange: "0.5",
+						LpPriceRange: "0.05",
```
Comment (jeremyletang) [inline proposals/market_perp_btcusd.go:156]: Shouldn't this be in the SLAParameter section?

File: proposals/market_perp_btcusd.go (hunk 2 above)
Comment (witgaw) [reply proposals/market_perp_btcusd.go]: Ok, went for 2mins and used 10x mainnet linear slippage factor

File: proposals/market_perp_btcusd.go (hunk 3 above)
Comment (witgaw) [reply proposals/market_perp_btcusd.go:156]: No idea, just modifying the existing setup, but it compiles so either it's using older protos or it is like that.

**Pull request merged**